Encounter1/RegActivity.cs

```diff
@@ -38,40 +40,123 @@ protected override void OnCreate(Bundle savedInstanceState)
             animationDrawable.Start();
 
             // Create your application here  
-            btnCreate = FindViewById<Button>(Resource.Id.button1);
-            txtUsername = FindViewById<EditText>(Resource.Id.editText2);
-            txtPassword = FindViewById<EditText>(Resource.Id.editText1);
-    
+            txtUsername = (EditText)FindViewById<EditText>(Resource.Id.editText2);
+            txtPassword = (EditText)FindViewById<EditText>(Resource.Id.editText1);
+            confirmPassword = (EditText)FindViewById<EditText>(Resource.Id.editText3);
+            txtEmail = (EditText)FindViewById<EditText>(Resource.Id.editText4);
+            btnCreate = FindViewById<Button>(Resource.Id.buttonCreate);
             btnCreate.Click += Btncreate_Click;
         }
-        private void Btncreate_Click (object sender, EventArgs e)
+
+        private Boolean RegistrationPassword()
         {
-            try
+            var userReg = (string)txtPassword.Text;
+
+            if (string.IsNullOrEmpty(userReg))
             {
-                
-                string dpPath = Path.Combine(System.Environment.GetFolderPath(System.Environment.SpecialFolder.Personal), "users.db3");
-                var db = new SQLiteConnection(dpPath);
-                db.CreateTable<LoginTable>();
-                LoginTable tbl = new LoginTable
-                {
-                    Username = txtUsername.Text,
-                    Password = txtPassword.Text
-                };
-              
-                db.Insert(tbl);
-                Toast.MakeText(this, "Record Added Successfully...,", ToastLength.Short).Show();
+                return false;
             }
-            catch (Exception ex)
+            else
             {
-                Toast.MakeText(this, ex.ToString(), ToastLength.Short).Show();
+                return true;
             }
-            StartActivity(typeof(MainActivity));
         }
 
-    }
+        private Boolean RegistrationConfirmPassword()
+        {
+            var userReg = confirmPassword.Text;
+
+            if (string.IsNullOrEmpty(userReg))
+            {
+                return false;
+            }
+            else
+            {
+                return true;
+            }
+        }
+
+        private Boolean Verify()
```

Comment:
Verify is very vague name. It might be better to change to VerifyPasswordMatchConfirmPassword